src/binary-reader-objdump.cc

```diff
@@ -500,6 +520,7 @@ class BinaryReaderObjdumpDisassemble : public BinaryReaderObjdumpBase {
   Offset last_opcode_end = 0;
   int indent_level = 0;
   Index next_reloc = 0;
+  Index function_index = 0;
```

Comment:
How about `current_function_index` to match the members above?

Reply:
Will do!